src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php

```diff
@@ -30,6 +31,16 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $builder->addModelTransformer(new ReversedTransformer(new ResourceToIdentifierTransformer($this->countryRepository, 'code')));
     }
 
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver
+            ->setDefaults([
+                'attr' => ['class' => 'country_search_dropdown ui fluid search selection dropdown'],
+                'enabled' => 'false',
```

Comment:
I can add here almost the same comment as in the previous iteration: https://github.com/Sylius/Sylius/pull/14058#discussion_r891936298

`CountryCodeChoiceType` is used in several different places if I'm correct, but you want to change the behaviour only for selecting countries for zones, so this parameter should be disabled only in `ZoneType`, at least for now

Reply:
I'm almost sure that if you will change this line: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AddressingBundle/Form/Type/ZoneType.php#L104 by adding there an `enabled` parameter, it will work correctly:
```
ZoneInterface::TYPE_COUNTRY => ['label' => 'sylius.form.zone.types.country', 'enabled' => false],
```
And you will not need to add a custom form type. Did you check this approach?